rpc/transport/nats/server.go

```diff
@@ -91,24 +84,6 @@ func (c *natsTransportServer) RegisterRequestHandler(apiVersion string, handler
 	return err
 }
 
-func (c *natsTransportClient) Subscribe() (<-chan []byte, error) {
-	if c.notificationChan != nil {
-		return c.notificationChan, nil
-	}
-	c.notificationChan = make(chan []byte)
-	subscription, err := c.nc.Subscribe(nitroNotificationTopic, func(msg *nats.Msg) {
-		c.notificationChan <- msg.Data
-	})
-	c.natsSubscriptions = append(c.natsSubscriptions, subscription)
-
-	return c.notificationChan, err
-}
-
-func (c *natsTransportClient) Close() {
-	c.natsTransport.Close()
-	close(c.notificationChan)
-}
-
```

Comment:
Thanks for tidying this up!

In an ideal world, we might split out this work into a separate PR -- especially since it is the kind of diff which looks more drastic than it actually is. 

I think it's fine to keep it here this time.